hi-ml-cpath/Makefile

```diff
@@ -81,6 +81,11 @@ pytest:
 pytest_coverage:
 	pytest --cov=health_cpath --cov SSL --cov-branch --cov-report=html --cov-report=xml --cov-report=term-missing --cov-config=.coveragerc
 
+# run pytest using GPUs via an AzureML job
+pytest_gpu:
+	python ../hi-ml-azure/run_pytest.py --mark=gpu --cluster=pr-gpu --conda_env=environment.yml --folder=hi-ml-cpath --coverage_module=health_cpath
```

Comment:
you might want to add --strictly_aml_v1=True here. (it's true by default but the docu is misleading, I updated it in #695)